finagle-core/src/test/scala/com/twitter/finagle/util/LoadServiceTest.scala

```diff
@@ -94,6 +95,12 @@ class LoadServiceTest extends FunSuite with MockitoSugar {
     executor.shutdown()
   }
 
+  test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
```

Comment:
test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
  ignoredPaths.let(Seq("com.twitter.finagle.util")){
      assert(LoadService[LoadServiceRandomInterface]().isEmpty)
   }
 }

I used a similar LoadService call as done for the Announcer above so i assumed this should ignore things within the package. However it did return an object. I even tried it with directory notation with "com/twitter/finagle/util/" with the same result.